src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -149,7 +179,14 @@ class PythonInferSubsystem(Subsystem):
             you should enable this option.
             """
         ),
+        removal_version="2.14.0.dev1",
+        removal_hint=(
```

Comment:
No `softwrap`? (Let's play "spot the spacing mistake" :wink: )

Reply:
old habits die hard hehe